scipy/stats/stats.py

```diff
@@ -1037,9 +1039,58 @@ def moment(a, moment=1, axis=0):
        is None. The denominator for the moment calculation is the number of
        observations, no degrees of freedom correction is done.
 
+    See also
+    --------
+    kurtosis, skew, describe
+
+    Notes
+    -----
+    [1]_ The n-th moment of a real-valued continuous function f(x) of a real
+    variable about a value c is:
+    .. math::
+
+    m_k = \frac{1}{n} \sum_{i = 1}^n (x_i - \bar{x})^k
```

Comment:
Some proofreading would be helpful: n-th moment or m_k, sum over I or k or m, "continuous function f(x)" has nothing to do with anything here, what is x-bar and what is a "value c".